core/src/main/java/com/scalar/db/storage/jdbc/JdbcConfig.java

```diff
@@ -47,6 +47,11 @@ public class JdbcConfig {
   public static final String ADMIN_CONNECTION_POOL_MAX_TOTAL =
       PREFIX + "admin.connection_pool.max_total";
 
+  public static final String MYSQL_VARIABLE_KEY_COLUMN_SIZE =
+      PREFIX + "mysql.variable_key_column_size";
+  public static final String ORACLE_VARIABLE_KEY_COLUMN_SIZE =
+      PREFIX + "oracle.variable_key_column_size";
```

Comment:
I introduced two separate settings (only) for MySQL and Oracle for the following reasons.

- PostgreSQL also has a limit of 10485760 for each key column (but not for total). So, users might be confused if the common setting `jdbc.variable_key_column_size` does not affect it.
- We can also make the limit of PostgreSQL configurable, but the common settings would decrease the current limit since 10485760 is too large for MySQL and Oracle.
- I guess there is no motivation to decrease 10485760 for PostgreSQL.

Reply:
What is the reasoning for adding `variable` in the configuration name? 
In essence, it is a configuration, implying that the value is configurable (== variable). Following the same logic, we could add `variable` in all the configuration names.

Reply:
@Torch3333 Thank you for the question. It's a good point. It doesn't mean it's configurable, but the size of the variable (data-type) key column. I would like to distinguish it from other fixed-size data types like `int` and `double` since this option only affects data types (e.g., `varchar` and `varbinary`) with variable length. Still weird?

Reply:
Oh, ok. I completely missed this interpretation. I agree with this reasoning. 👍 
Thank for the explanation.